### ISSUE TITLE
Gender pronouns

### ISSUE THREAD

**Issue by jachwe** [berkmancenter/namae#45]
Hi. First, thank you for this great project. It helped me already a lot.
I now, for the first time, came across an issue:
When people add gender pronouns (e.g. `(he/him)`) to their names, the parsing fails all together. 

Example:
```
2.7.4 :003 > Namae.parse("Max Power (he/him, er/ihm)")
 => []
2.7.4 :004 > Namae.parse("Max Power (er/ihm)")
 => []
2.7.4 :005 > Namae.parse("Max Power")
 =>
[#<struct Namae::Name
  family="Power",
  given="Max",
  suffix=nil,
  particle=nil,
  dropping_particle=nil,
  nick=nil,
  appellation=nil,
  title=nil>]
```
I read about adding custom suffices, but then i though that this might become it's own part.
Currently i am trying to build that in and do a PR, but it goes slowely.

So very open for hints or other opinions.

**Comment by inukshuk** [comment]
Good point! Yes, I think pronouns should be their own category, because they could be given in addition to a suffix. They'd either have to be detected using syntax or by using a pattern featuring the actual pronouns; which I guess would be OK because the patterns can be adjusted at runtime so it's no big deal to add support for more languages if yours isn't covered yet.

**Comment by jachwe** [comment]
Here is a PR for this.
#46

**Comment by inukshuk** [comment]
Awesome, thanks! I'll take a look at it as soon as possible.
